Makefile

```diff
@@ -1,6 +1,7 @@
 .PHONY: build prepare run test seed down setup-apps
 
-COMPOSE = docker-compose -f compose/app.yaml -f compose/backend.yaml
+WB_ENV = development
```

Comment:
I like the idea to have WORKBENCH_ENV=development/test
but not production it can be confusing

also we don't really need the feature of shared volume at the moment.